test/behaviour/connection/database/BUILD

```diff
@@ -41,28 +41,27 @@ java_library(
     ],
 )
 
-grakn_behaviour_java_test(
-    name = "test",
+grakn_java_test(
+    name = "test-core",
```

Comment:
We need to split up this target in order to specify different `@ignore` tags for each.